fluentd/generate_throttle_configs.rb

```diff
@@ -102,8 +102,9 @@ def create_default_docker(excluded)
 
   file_name = "/etc/fluent/configs.d/input/docker/default-docker.conf"
 
-  File.open(file_name, 'w') { |file|
-    file.write(<<-CONF)
+  unless ENV['USE_JOURNAL'] == "true"
```

Comment:
so using Journal replaces the use of the default Docker config?  what bout if the user is trying to also throttle, would they end up with a combination of journal and in_tail?